src/extensions/default/CodeFolding/unittest-files/test.html

```diff
@@ -0,0 +1,38 @@
+<html>
+    <head>
+    
+    </head>
+    <body>
+   
+        <form action="#" method="post">
+            <div>
+                 <label for="name">Text Input:</label>
+                 <input type="text" name="name" id="name" value="" tabindex="1" />
+            </div>
+            <table>
+                <thead>
+                    <tr>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td></td>
+                    </tr>
+                </tbody>
+            </table>
+            <div>
+                 <h4>Radio Button Choice</h4>
+
+                 <label 
+                        for="radio-choice-1">
+                     Choice 1
+                </label>
+                 <input type="radio" name="radio-choice-1" id="radio-choice-1" tabindex="2" value="choice-1" />
+
+                 <label for="radio-choice-2">Choice 2</label>
+                 <input type="radio" name="radio-choice-2" id="radio-choice-2" tabindex="3" value="choice-2" />
+            </div>
+        </form>
+    </body>
+</html>
```

Comment:
Add a newline at the end of file please.
Are the empy lines necessary? If not remove.